lintcheck/src/main.rs

```diff
@@ -398,16 +399,9 @@ impl Crate {
             for opt in options {
                 clippy_args.push(opt);
             }
-        } else {
-            clippy_args.extend(["-Wclippy::pedantic", "-Wclippy::cargo"]);
         }
```

Comment:
Does this mean setting options in the sources `.toml` would no longer override the defaults? tbh we could probably remove options, I don't see a particular reason to use it over `--filter` etc

Reply:
Tbh, I don't quite know why this is here. The current toml file doesn't seem to use it. It could be useful to enable/disabled features, but even then I'd think that the lint levels should be independent of it.

So I would keep the option, but decouple the lint levels, like I did in this PR

Reply:
It was documented as a way of setting per crate lint levels - https://github.com/rust-lang/rust-clippy/tree/master/lintcheck#command-line-options-optional

I've never used it though, so changing or even removing it is fine by me but we'd want to update those docs either way

Reply:
I would keep it for the option to change the default features.

Good point with the documentation. I forgot about that :D

Reply:
I updated the documentation and also added a note about sandboxing or lack thereof. With the top 200 crates, this should be fine, but it's still something users should be aware of IMO